core-feature-pack/src/main/resources/content/bin/standalone.conf

```diff
@@ -81,3 +81,6 @@ fi
 
 # Uncomment this out to control garbage collection logging
 # GC_LOG="true"
+
+# Uncomment and edit to use a custom java.security file to override all the Java security properties
+#JAVA_OPTS="$JAVA_OPTS -Djava.security.properties==/path/to/custom/java.security"
```

Comment:
Is the double equal meant to be there?

Reply:
Yes, the double equals are needed to make sure that the java.security file is completely overridden.